scripts/format

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -eufo pipefail
+
+echo "goimports" >&2
```

Comment:
I like the idea, but I don't think I see it running on CI for that repo to check that it was actually done anywhere? Feels like an _additional_ rather than an _instead_?